po/de.po

```diff
@@ -1243,7 +1286,7 @@ msgstr "Farbiger Text"
 
 #: setzer/document/autocomplete/autocomplete.py:585
 msgid "Monospaced Text"
-msgstr ""
+msgstr "nichtproportionaler Text"
```

Comment:
I had to look it up but that's actually the German word for it: https://de.wikipedia.org/wiki/Nichtproportionale_Schriftart

Reply:
I would change that to "Monospace-Text".